wdio.conf.js

```diff
@@ -47,10 +48,30 @@ const chromeCapabilities = {
   }
 };
 const firefoxCapabilities = {
-  browserName: 'firefox'
+  browserName: 'firefox',
+  'moz:firefoxOptions': {
+    args: [
+      '-start-debugger-server',
```

Comment:
What is this needed for?

Reply:
Moved down to saucelabs, this enables extendedDebugging